packages/mobile/src/account/reducer.ts

```diff
@@ -294,6 +296,12 @@ export const reducer = (
         finclusiveKycStatus: action.finclusiveKycStatus,
       }
     }
+    case Actions.SET_FINCLUSIVE_REGION_SUPPORTED: {
+      return {
+        ...state,
+        finclusiveRegionSupported: true,
```

Comment:
shouldn't it be possible to set this to false, too? like with `finclusiveRegionSupported: action.finclusiveRegionSupported`?

Reply:
the default is false. the prop can only be set to true during the persona onSuccess callback.

Reply:
Yes I understand that, I was suggesting that we make it possible to set to either true or false.

But whatever, I don't care that much